apps/kyb-app/src/components/organisms/UIRenderer/hooks/useUIElementHandlers/useUIElementHandlers.ts

```diff
@@ -15,10 +15,11 @@ export const useUIElementHandlers = (definition: UIElement<AnyObject>) => {
       const context = getContext();
 
       //@ts-ignore
-      set(context, definition.valueDestination || '', event.target.value as any);
+      set(context, definition.valueDestination || '', event.target.value);
 
       setContext(context);
       emitEvent('onChange');
+      debugger;
```

Comment:
👀

Reply:
Will be cleaned in destination branch, not quite related to backoffice in this case